.drone.yml

```diff
@@ -41,7 +41,7 @@ unit_tests: &unit_tests
 
 sonar_scanner: &sonar_scanner
   pull: if-not-exists
```

Comment:
if it doesn't work just add a comment saying usually good practice is to have a version number a sha pointing to the image because you don't want to grab the latest one in case it gets updated